Apps/VisionOS/VisionOS/TutorialViews/HowToTestIdentifyTutorialView.swift

```diff
@@ -0,0 +1,51 @@
+import MarkdownUI
+import SwiftUI
+
+struct HowToTestIdentifyTutorialView: View {
+    static let title = ScreenTitleConfig("How to Test Identify")
+
+    @EnvironmentObject private var viewModel: ViewModel
+
+    var body: some View {
+        ScrollView(.vertical) {
+            VStack(alignment: .leading) {
+                Markdown {
+"""
+You can follow the steps below to verify whether your setup and
+your use of `CustomerIO.shared.identify` works as expected.
+
+Assuming you called identify with an identifier + an email.
+
+1. Go to your
+[workplace dashboard](https://fly.customer.io/workspaces/dashboard)
```

Comment:
Do you mean workspace here ? AFAIK, we have never used the term `workplace` while referring to Fly ! Feel free to correct me if I am mistaken !

Reply:
you are absolutely right! this is a typo